container_api/src/config.rs

```diff
@@ -542,4 +629,137 @@ pub mod tests {
             panic!("Should have failed!")
         }
     }
+
+    fn bridges_config(bridges: &str) -> String {
+        format!(
+            r#"
+    [[agents]]
+    id = "test agent"
+    name = "Holo Tester 1"
+    public_address = "HoloTester1-------------------------------------------------------------------------AHi1"
+    key_file = "holo_tester.key"
+
+    [[dnas]]
+    id = "app spec rust"
+    file = "app-spec-rust.hcpkg"
+    hash = "Qm328wyq38924y"
+
+    [[instances]]
+    id = "app1"
+    dna = "app spec rust"
+    agent = "test agent"
+    [instances.logger]
+    type = "simple"
+    file = "app_spec.log"
+    [instances.storage]
+    type = "file"
+    path = "app_spec_storage"
+
+    [[instances]]
+    id = "app2"
+    dna = "app spec rust"
+    agent = "test agent"
+    [instances.logger]
+    type = "simple"
+    file = "app_spec.log"
+    [instances.storage]
+    type = "file"
+    path = "app_spec_storage"
+
+    [[instances]]
+    id = "app3"
+    dna = "app spec rust"
+    agent = "test agent"
+    [instances.logger]
+    type = "simple"
+    file = "app_spec.log"
+    [instances.storage]
+    type = "file"
+    path = "app_spec_storage"
+
+    {}
```

Comment:
what's this?

Reply:
This whole string block is in a `format!`, so this is where the parameter gets inserted. The following tests just change the defined bridges so I wanted to dry these tests out a bit at least.. could be more..